apis/Google.Cloud.Spanner.Data/Google.Cloud.Spanner.Data/SpannerParameterCollection.cs

```diff
@@ -230,11 +230,12 @@ private void FillSpannerInternalTypes(MapField<string, V1.Type> typeDictionary)
             }
         }
 
-#if NET45 // It's odd that these are required, but it looks like it's due to a discrepancy
-// between the reference assemblies and real assemblies. See
-// https://stackoverflow.com/questions/44197176 for details.
-// Fortunately the real implementations all just return false too.
-/// <inheritdoc />
+#if NET45
+        // It's odd that these are required, but it looks like it's due to a discrepancy
```

Comment:
We should probably remove this comment now - it looks like it's *actually* a difference between .NET 4.5 and 4.5.1, and the docs being wrong. (I think the old project.json version used "whatever .NET 4.5 version you had installed" rather than the reference assemblies.)